test/verify/check-machines-dbus

```diff
@@ -1849,8 +1853,21 @@ class TestMachinesDBus(machineslib.TestMachines):
             name="my_logical_pool",
             pool_type="logical",
             source={"source_path": "vol_grp1"},
+            remove=False,
+        ).execute()
+
+        # XFAIL: Try to create a pool with already existing source
```

Comment:
Why "XFAIL"? That's what this PR is supposed to fix, no?

Reply:
He added an inline error in the dialog, in case the user tried to create a LVM pool with the same source as an existing pool, this XFAIL.

Anyway, I think we should not do that, see comment bellow.